### ISSUE TITLE
Improve classification accuracy in constrast to localization

### ISSUE THREAD

**Issue by lleye** [ultralytics/yolov5#5300]
Hello,

Context: a model does a better job in localization (exact bounding boxes) but a relatively bad job on classification. To increase the tradeoff between classification and bounding boxes in favor of classification, do I increase or reduce the cls loss gain? Also is there a better/programatic way to do this? e.g. use evolution?

@glenn-jocher

**Comment by glenn-jocher** [comment]
@lleye yes if you have resources and time then evolution is the automated way to solve for optimal hyperparameters. You can also try rebalancing your losses manually as you say by raising or decreasing your classification loss gain. Whether to raise or decrease it depends on your level of classification overfitting and other factors, but to be honest no one really has any idea what to do there, guidance is usually simply obtained from empirical results, i.e. try it both ways and go with what works.

**Comment by lleye** [comment]
thanks for the reply. Will try both and try to report back on findings.
